src/llfs/volume.cpp

```diff
@@ -572,7 +572,10 @@ StatusOr<SlotRange> Volume::append(AppendableJob&& appendable, batt::Grant& gran
 
     const auto grant_size_after_prepare = grant.size();
 
-    BATT_CHECK_EQ(prepared_job_size, grant_size_before_prepare - grant_size_after_prepare);
+    if (prepare_slot.ok()) {
+      BATT_CHECK_EQ(prepared_job_size, grant_size_before_prepare - grant_size_after_prepare)
+          << BATT_INSPECT(prepare_slot.status()) << BATT_INSPECT(grant_size_before_prepare);
+    }
 
```

Comment:
We are going to fail the operation on line 590 (`BATT_REQUIRE_OK(prepare_slot);`)... if verbose logging is on, this will emit a log message.  It's not always unexpected behavior; for example, this might happen when shutting down  the system.